session-notes/Co-Authored-By_count_after_cleanup

```diff
@@ -0,0 +1,27 @@
+  43 ## Co-Authors
```

Comment:
Are those files necessary?

Reply:
No, but I wanted some way to show results.

Suggestions for a better place to archive the quick-and-dirty data?